docs/app/routes/docs/guides/react-three-fiber.mdx

```diff
@@ -0,0 +1,401 @@
+---
+meta:
+  title: R3F | React Spring
+  'og:title': R3F | React Spring
+  'twitter:title': R3F | React Spring
+  description: A detailed guide to using React Spring with R3F.
+  'og:description': A detailed guide to using React Spring with R3F.
+  'twitter:description': A detailed guide to using React Spring with R3F.
+  'og:url': https://beta.react-spring.dev/docs/guides/react-three-fiber
+  'twitter:url': https://beta.react-spring.dev/docs/guides/react-three-fiber
+sidebar_position: 1
+---
+
+# React Three Fiber
+
+:::note
+It's assumed you have a base understanding of `react-spring` and `react-three-fiber`. If you're new to
+either, check out our [getting started](/docs/getting-started) or alternatively, the [r3f](https://docs.pmnd.rs/react-three-fiber) docs.
+:::
```

Comment:
Nit, you'll want to consistently spell or refer to R3F or react-three-fiber here. @hmans has asked before about how to refer to R3F as a package -- whether that's `@react-three/fiber`, react-three-fiber, R3F, etc.

Reply:
Really good point.